standalone/api.c

```diff
@@ -782,3 +786,61 @@ void modsecReportRemoteLoadedRules()
 
 }
 #endif
+
+#ifdef WAF_JSON_LOGGING_ENABLE
+void modsecReopenLogfileIfNeeded(request_rec *r)
+{
+    modsec_rec *msr = NULL;
+    int rc = 0;
+    apr_file_t * fd = NULL;
+    /* Find the transaction context first. */
+    msr = retrieve_msr(r);
+
+    if (msr == NULL)
+        return;
+
+    if (msc_waf_log_reopened){
```

Comment:
the name `msc_waf_log_reopened` makes it sound like the file was already reopened, but I think it was not at this point. Would it make more sense to call it something like `msc_waf_log_reopen_requested` or `msc_waf_log_should_reopen`?